### ISSUE TITLE
partkeepr after night stop to work 0.81

### ISSUE THREAD

**Issue by mctomaszek** [partkeepr/PartKeepr#780]
partkeepr without obvious reason stopped to work. login page does not come in. i dont known what is happened. i have tray to run setup but looks like is a problem with php; does not want to pass testing for correct php settings; server is responding "invalid response from server". could you help me please. i did not touch a server and settings.....

regards

**Comment by kpostrup** [comment]
I have the same problem, with PartKeepr 1.0, stopped working overnight, but has been running 1.0 for months without issues.

I've tried to re-run setup, and everything passes just fine. The login prompt doesn't show.
Using the browser inspector, I get the following error:

> Uncaught Error: Module: 'feeds' not found!
>     at ?file=feeds&v=1.x&output=nocss%3Dtrue&async=2:1
> (anonymous) @ ?file=feeds&v=1.x&output=nocss%3Dtrue&async=2:1
> 

I hope this helps.

**Comment by gustavovelascoh** [comment]
I have the same problem, and it happens while using both v1.1.0 and last commit.

**Comment by lazosweb** [comment]
Same for mine.

Error: Module: 'feeds' not found!  uds:1:17
	<anonymous> https://www.google.com/uds/:1:17

**Comment by Gotos** [comment]
Can someone with this problem please run the unit tests and post the output? That might help.
https://wiki.partkeepr.org/wiki/Developers/Unit_Testing (Just running the first two commands on that page should be fine)

**Comment by gustavovelascoh** [comment]
Could be related with this?

http://stackoverflow.com/questions/41592997/alternate-possible-solution-for-google-feeds-api

**Comment by lazosweb** [comment]
It might be, but all we have to do now is be patient and wait for the developer to look into it.

**Comment by felicitus** [comment]
The issue is caused since in earlier PartKeepr versions we include all user extensions for ExtJS, which in turn remote load code from Google Servers, which now seems to be gone.

You should be able to fix this by opening the file `src/PartKeepr/FrontendBundle/Resources/views/index.html`

Find the 2 lines containing:
```'
js/packages/extjs6/build/packages/ux/classic/ux-debug.js' %}
```

and replace each line it with:

```
        'js/packages/extjs6/packages/ux/classic/src/TreePicker.js'
        'js/packages/extjs6/packages/ux/classic/src/TabCloseMenu.js'
        'js/packages/extjs6/packages/ux/classic/src/statusbar/StatusBar.js'
        'js/packages/extjs6/packages/ux/classic/src/IFrame.js' %}
```

Re-run setup and check if it works now.

**Comment by felicitus** [comment]
@gustavovelascoh the issue should not occur with 1.1.0 or master; can you double-check that when you tested 1.1.0 or master that you have indeed cleared your browser's cache and re-ran setup?

**Comment by lazosweb** [comment]
Ok I am on 1.0.0 according to the change log. I will update it and see if the 1.1.0 wont have this problem.

**Comment by gustavovelascoh** [comment]
@seisvalt "@gustavovelascoh the issue should not occur with 1.1.0 or master; can you double-check that when you tested 1.1.0 or master that you have indeed cleared your browser's cache and re-ran setup?"

**Comment by lazosweb** [comment]
@felicitus Mine still does not work. I have updated partkeepr from 1.0.0 to 1.1.0 I cleared the browser's cache and I still have the same problem. Before you wrote re-ran setup. Is that mandatory. If we do that it will overwrite the existing database?

**Comment by mctomaszek** [comment]
try to use another browser

**Comment by twelve12pm** [comment]
Other than google APIs, are there any other dependencies that live somewhere on the web? Our PartKeepr was working perfectly but suddenly quit due to this issue. If it happened once, it will probably happen again in the future. Is it possible to make a private copy of all dependencies in our PartKeepr server? PartKeepr needs to be timeless!

**Comment by felicitus** [comment]
Not that we are aware of. That's why the issue was fixed in PartKeepr 1.1.0, we simply weren't aware of the issue previously.
